lib/super_good/solidus_taxjar.rb

```diff
@@ -1,3 +1,4 @@
+require "deface"
```

Comment:
We may need to move this after `solidus_core` 🤔 It looks like Rails is required but not loaded yet at this point in time.

Reply:
Ah, thank you. Resolved.